resources/views/api/emails/voucher_payrequest_email.blade.php

```diff
@@ -3,9 +3,9 @@
 @section('content')
 
     <section role="main">
-        <h1 style="text-align: center;">Voucher Payment Request</h1>
+        <h1>Voucher Payment Request</h1>
 
-        <p>[xXx]Hi {{ config('mail.to_admin.name') }},</p>
+        <p>Hi {{ config('mail.to_admin.name') }},</p>
```

Comment:
Since we use 'hi' everywhere else, I'll just change the one place where it uses 'hello' to 'hi' if that's okay.